src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt

```diff
@@ -128,6 +136,13 @@ class PaymentRequestsService(
       return@flatMap paymentRequestInfo
     }
 
+  private fun traceVerifyNodoError(faultCode: String?) {
+    openTelemetryUtils.addErrorSpanWithAttributes(
+      VERIFY_PAYMENT_NOTICE_NODO_ERROR_SPAN_NAME,
+      Attributes.of(
+        AttributeKey.stringKey(FAULT_CODE_SPAN_KEY), Optional.ofNullable(faultCode).orElse("null")))
```

Comment:
```suggestion
        AttributeKey.stringKey(FAULT_CODE_SPAN_KEY), Optional.ofNullable(faultCode).orElse("No faultCode received")))
```
Use here the same default string used for transactions-service